frontend/src/components/design-system/layout/StackComponent.vue

```diff
@@ -0,0 +1,51 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+
+export interface Props {
+  horizontal?: boolean;
+  spacing?: 'extra-small' | 'small' | 'medium' | 'large';
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  horizontal: false,
+  spacing: 'small',
+});
+
+const classes = computed(() => ({
+  'stack--horizontal': props.horizontal,
+  [`stack--${props.spacing}`]: props.spacing,
+}));
+</script>
+
+<template>
+  <div :class="['stack', classes]">
+    <slot />
+  </div>
+</template>
+
+<style scoped lang="scss">
+.stack {
+  display: flex;
+  flex-direction: column;
+
+  &--horizontal {
+    flex-direction: row;
+  }
+
+  &--extra-small:deep() * + * {
```

Comment:
Ha, sova. Obávám se ale, že by tam měl být jestě `>` selektor (`&--extra-small:deep() > * + *`). Přijde mi, že takto asi přidáváš odsazení na všech úrovních zanoření..? Nezkoušel jsem to, tak jestli ty jo, tak dobrý.

Reply:
Jo, good point, asi jo. Opravuju.